umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj

```diff
@@ -239,4 +239,6 @@
 (defmethod interface/migrate-umm-version [:collection "1.10" "1.9"]
   [context c & _]
   (-> c
-      coll-progress-migration/migrate-down))
+      coll-progress-migration/migrate-down
+      (util/update-in-all [:RelatedUrls :GetData] dissoc :MimeType)
+      (util/update-in-all [:RelatedUrls :GetService] dissoc :Format)))
```

Comment:
There is umm_spec/test/migration/version/collection.clj used to test the migration. I didn't see it.

Reply:
Done.